### PR TITLE
Remove CentOS 8 from Make targets and documentation

### PR DESCRIPTION
**Pull request by cbandy**
CentOS 8 reached [End of Life](https://www.centos.org/centos-linux-eol/) (EOL) on Dec 31, 2021.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [x] Documentation
 - [x] Other

**Other Information**:

Issue: [sc-13732]

### CODE REVIEW

File: testing/kuttl/README.md [hunk 1]
```diff
@@ -62,28 +65,28 @@ change those K8s objects before applying them. That means that, if we wanted to
...
-in `source` template YAML (the `e2e` and `e2e-other` folders) and writing those files to the
-`testing/kuttl/e2e-generated` and `testing/kuttl/e2e-generated-other` folders.
+and writing those files to the `testing/kuttl/e2e-generated*` directories.
 
 These templated test files can be generated by setting some variables in the command line and
 calling the `make generate-kuttl` target:
 
 ```console
-KUTTL_PG_VERSION=13 KUTTL_POSTGIS_VERSION=3.0 KUTTL_PSQL_IMAGE=registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-0 make generate-kuttl
+KUTTL_PG_VERSION=13 KUTTL_POSTGIS_VERSION=3.0 make generate-kuttl
```
Comment (ValClarkson) [inline testing/kuttl/README.md:74]: I like leaving the KUTTL_PSQL_IMAGE in the example here which shows how you can easily set an image to use when running via the command line.

**Review by ValClarkson — APPROVED** [review]
Just the one comment, lgtm otherwise

File: testing/kuttl/README.md (hunk 1 above)
Comment (cbandy) [reply testing/kuttl/README.md:74]: It's set the same way as these other variables. We shouldn't need to change it when testing different versions.

Do you think we should put every variable in this readme, or leave it to the target for now?

https://github.com/CrunchyData/postgres-operator/blob/2b2e019e117ab1ce80cf16cf138e22e1039c14ad/Makefile#L236-L240

**Comment by cbandy** [comment]
Rebased.

**Pull request merged**